packages/react-server/src/render/test/render.test.tsx

```diff
@@ -24,11 +24,24 @@ describe('createRender', () => {
     const ctx = createMockContext();
 
     const renderFunction = createRender(() => <>{myCoolApp}</>);
-    await renderFunction(ctx as RenderContext);
+    await renderFunction(ctx);
 
     expect(ctx.body).toContain(myCoolApp);
   });
 
+  // we cannot test this with `createMockContext` because it actually clobbers the proxy methods itself :(
```

Comment:
😂

Reply:
Is there an issue for this? Seems... problematic

Reply:
updated the PR, I was just doing it wrong and am a dumdum